scrapper.py

```diff
@@ -35,7 +46,31 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'user-agent': "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36"
+        }
+
+        sleep_period = 5
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+            time.sleep(sleep_period)
+
+            if not response.ok:
+                print("Request failed")
+
+            with open(ASSETS_PATH / "index.html", 'w', encoding='utf-8') as file:
```

Comment:
we should not save any html files as a part of this assignment. especially all in one file, rewriting it each time